packages/cli/src/lingui-compile.ts

```diff
@@ -35,23 +44,25 @@ function command(config: LinguiConfig, options) {
   const doMerge = !!config.catalogsMergePath
   let mergedCatalogs = {}
 
-  console.error("Compiling message catalogs…")
+  console.log("Compiling message catalogs…")
 
-  config.locales.forEach((locale) => {
+  for (const locale of config.locales) {
```

Comment:
Changed to the loop to be able to "early return" from function. 

`process.exit(1)` used previously for early return is not test-friendly